slack_driver/serverless.yml

```diff
@@ -0,0 +1,97 @@
+service: slack-driver
+plugins:
+  - serverless-python-requirements
+custom:
+  stage: ${env:STAGE}
+  pythonRequirements:
+    dockerizePip: false
+    dockerImage: mozillaiam/docker-sls:latest
+    cleanupZipHelper: true
+provider:
+  name: aws
+  runtime: python3.6
+  stage: ${env:STAGE}
+  iamRoleStatements:
+    - Effect: "Allow"
+      Action:
+        - "dynamodb:DescribeTable"
+        - "dynamodb:DescribeStream"
+        - "dynamodb:GetItem"
+        - "dynamodb:GetRecords"
+        - "dynamodb:GetShardIterator"
+        - "dynamodb:ListStreams"
+        - "dynamodb:ListTables"
+        - "dynamodb:ListTagsOfResource"
+        - "dynamodb:Query"
+        - "dynamodb:Scan"
+      Resource: arn:aws:dynamodb:*:*:table/cis-stream-prod-and-idv-IdentityVaultUsers-LMGGZ2XE8K6F
```

Comment:
The ARNs for this should come from your config files.

Reply:
note: this is from gsuite driver

Reply:
done